### PR TITLE
Web boxes website redesign

### PR DESCRIPTION
**Pull request by JoshAtticus**
Use an eye-pleasing design to create a clean, elegant-looking website, previews coming soon!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
 }
 @font-face{
 	font-family:'Simvoni';
-	src:url(./fonts/Simvoni.ttf) format("truetype")
+	src:url(./fonts/Simvoni-Bold.ttf) format("truetype")
```
Comment (JoshAtticus) [inline style.css:10]: Titles on iOS now look awful thanks to this, but there is no patch because iPads appear as Macs.

**Review by showierdata9978 — APPROVED** [review]
Looks great on mobile!

**Comment by JoshAtticus** [comment]
> Looks great on mobile!

Thanks!

**Comment by JoshAtticus** [comment]
@MikeDev101 I need you to approve my legal page changes.

**Pull request merged**